manifests/samples/full_stack_fs.yaml

```diff
@@ -73,6 +73,8 @@ spec:
     enabled: true
   portal: {}
   registry: {}
+  trivy:
```

Comment:
don't add trivy in this file, it should in full_stack_other_fs.yml

Reply:
Why can't we enable trivy in this manifest?